src/components/nodes/gateway/gateway.vue

```diff
@@ -42,6 +42,11 @@ export default {
     'isRendering',
   ],
   mixins: [highlightConfig, portsConfig, hideLabelOnDrag],
+  created() {
+    const flow = this.node.definition.default || null;
+    delete this.node.definition.default;
```

Comment:
I'm a little confused why you're deleting and re-adding it - could you give some overview why this needs to be done this way?

Reply:
Is to fix a weird issue when loading a process with a gateway with a defined "default" property, it is not reactive, then the user can not change the default flow